<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1312</width>
    <height>792</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="manID">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>450</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="serNO">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="status">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>400</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="eng">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
   </widget>
   <widget class="QTextBrowser" name="pow">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
   </widget>
   <widget class="QTextBrowser" name="voltage">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame{border:1px solid rgb(0,255,255)}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLabel" name="clock">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1311</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(211, 215, 207);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:26pt; color:#cc0000;&quot;&gt;PTB DC Meter&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="current">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="pubKey">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="cost">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>500</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="duration">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>550</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="eng_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>100</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:26pt; color:#cc0000;&quot;&gt;DC Meter 1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="pow_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>150</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
   </widget>
   <widget class="QTextBrowser" name="voltage_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>200</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame{border:1px solid rgb(0,255,255)}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
   </widget>
   <widget class="QTextBrowser" name="serNO_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>250</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="eng_3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>400</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
   </widget>
   <widget class="QTextBrowser" name="pow_3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>450</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
   </widget>
   <widget class="QTextBrowser" name="voltage_3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>500</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame{border:1px solid rgb(0,255,255)}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
   </widget>
   <widget class="QTextBrowser" name="serNO_3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>550</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>330</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:26pt; color:#cc0000;&quot;&gt;DC Meter 2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="eng_4">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>190</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
   </widget>
   <widget class="QTextBrowser" name="pow_4">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>240</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
   </widget>
   <widget class="QTextBrowser" name="voltage_4">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>290</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame{border:1px solid rgb(0,255,255)}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
   </widget>
   <widget class="QTextBrowser" name="serNO_4">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>340</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>120</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:26pt; color:#cc0000;&quot;&gt;DC Meter 1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <zorder>pow</zorder>
   <zorder>manID</zorder>
   <zorder>serNO</zorder>
   <zorder>status</zorder>
   <zorder>voltage</zorder>
   <zorder>clock</zorder>
   <zorder>label_8</zorder>
   <zorder>eng</zorder>
   <zorder>current</zorder>
   <zorder>pubKey</zorder>
   <zorder>cost</zorder>
   <zorder>duration</zorder>
   <zorder>eng_2</zorder>
   <zorder>label_9</zorder>
   <zorder>pow_2</zorder>
   <zorder>voltage_2</zorder>
   <zorder>serNO_2</zorder>
   <zorder>eng_3</zorder>
   <zorder>pow_3</zorder>
   <zorder>voltage_3</zorder>
   <zorder>serNO_3</zorder>
   <zorder>label_10</zorder>
   <zorder>eng_4</zorder>
   <zorder>pow_4</zorder>
   <zorder>voltage_4</zorder>
   <zorder>serNO_4</zorder>
   <zorder>label_11</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1312</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
